pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "vtpserver"
+version = "0.1.0"
+description = "A Python API server to demonstrate the use of VoteTrackerPlus for a mock election"
+authors = ["Neil Johnson <neil@cadent.com>"]
+license = "GNU General Public License v2.0"
+readme = "README.md"
+packages = [{include = "vtpserver", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.11"
```

Comment:
👉🏼 Important 👈🏼

Unless we have an explicit need for features in the latest Python release, we need to support earlier releases, definitely back to the most recent Ubuntu LTS (22.04) and ideally back at least a couple of years (if not all the way back to previous Ubuntu LTS (20.04)). I recommend this be no higher than 3.10, and ideally also consider supporting 3.9 which is going to still be in use for two more years (and past the date of the next national election).

This is not just a "nice-to-have": VTP is an unusual project in that the software meant to be run by regular users not just election administrators. While this is the server and not the VTP core tool (for which this is critical) if it's successful we should expect people to also look at the rest of the VTP projects. We can't require that people have to install their own Pythons beyond what's on the system to do it.

See:
- https://endoflife.date/python
- https://packages.ubuntu.com/jammy/python3
- https://packages.ubuntu.com/focal/python3